lib/src/bootloader.rs

```diff
@@ -13,14 +15,39 @@ pub(crate) fn install_via_bootupd(
     rootfs: &Utf8Path,
     configopts: &crate::install::InstallConfigOpts,
 ) -> Result<()> {
+    let device_str: &str;
+    let prepboot_dir;
     let verbose = std::env::var_os("BOOTC_BOOTLOADER_DEBUG").map(|_| "-vvvv");
     // bootc defaults to only targeting the platform boot method.
     let bootupd_opts = (!configopts.generic_image).then_some(["--update-firmware", "--auto"]);
+
+    if cfg!(target_arch = "powerpc64") {
+        // get PowerPC-PReP-boot device information
+        let result = Command::new("lsblk")
+            .args([
+                "--noheadings",
+                "--paths",
+                "--filter",
+                &(format!(r#"'PARTLABEL=="PowerPC-PReP-boot"'"#)),
```

Comment:
I think the problem may be here in that you're passing an extra `'` that is needed only when invoking from shell script.